lwr_controllers/src/dynamic_sliding_mode_controller.cpp

```diff
@@ -157,7 +41,7 @@ namespace lwr_controllers
 		k_.resize(kdl_chain_.getNrOfJoints());
 
 		sub_command_ = nh_.subscribe("command_configuration", 1, &DynamicSlidingModeController::command_configuration, this);
```

Comment:
Can you change this to
```c++
nh_.subscribe("command", 1, &DynamicSlidingModeController::command, this);
```
to be consistent with all controllers? It requires to change the function name.
Thanks.